site/styles/_pages.scss

```diff
@@ -3,6 +3,8 @@
   @include media-breakpoint-up(md){
     padding-bottom: $grid-gutter-width;
   }
+  flex: 1 0 auto;
```

Comment:
I believe [this](https://github.com/philipwalton/solved-by-flexbox/blob/master/assets/css/components/site.css#L21) should be just `flex: 1;` (as per [example](https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/)) with `0 auto` redundant because of the aforementioned browser fixes.

Reply:
will change in next commit